support/hololens/ServoApp/BrowserPage.cpp

```diff
@@ -59,17 +59,17 @@ void BrowserPage::BindServoEvents() {
       [=](hstring title, hstring artist, hstring album) {});
   servoControl().OnMediaSessionPlaybackStateChange(
       [=](const auto &, int state) {
-        if (state == servo::Servo::MediaSessionPlaybackState::None) {
+        if (state == static_cast<int>(servo::Servo::MediaSessionPlaybackState::None)) {
```

Comment:
can we not pass in a `Servo::MediaSessionPlaybackState` instead of an `int` here?

Reply:
`OnMediaSessionPlaybackStateChange` is a IDL method. It's possible but not straight forward (using Inspectable objects).

Reply:
Ah, okay, seems fine then